### ISSUE TITLE
[apex] System.Database.action() does not flag CRUD

### ISSUE THREAD

**Issue by nwcm** [pmd/pmd#4526]
**Affects PMD Version:** 7.0.0-rc2

**Rule:** ApexCRUDViolationRule

Please provide the rule name and a link to the rule documentation:
https://docs.pmd-code.org/latest/pmd_rules_apex_security.html#apexcrudviolation

**Description:**

**Code Sample demonstrating the issue:**

```java
System.Database.upsert(record);
```

**Expected outcome:**

PMD should report a violation at line ..., but doesn't. This is a false-negative.

**Running PMD through:** *[CLI]*
